### ISSUE TITLE
Graphs hard to parse for color blind people

### ISSUE THREAD

**Issue by SeanPedersen** [erikbern/ann-benchmarks#210]
Hey very cool project! I find the performance graphs really hard to understand though because of my red-green color blindness. I can imagine many others with worse color blindness having even worse issues. Is there a textual comparison table available? This would help a lot.

Another option could be to make the diagrams interactive and highlight the current selected algorithms performance in some way.

**Comment by erikbern** [comment]
There's actually interactive graphs available here! http://ann-benchmarks.com/glove-100-angular_10_angular.html

I'm not colorblind but honestly I find it hard to parse too. Maybe we could limit it to a smaller number of algorithms

**Comment by SeanPedersen** [comment]
Cool looks like I made a good choice using https://github.com/nmslib/hnswlib/ for my project.

Yes maybe reduce to the top 10 and the interactive graph is awesome, even for me! I suggest linking to it more prominently in the top section of the README as I was not aware of it after spending some time looking for it.

**Comment by lc0** [comment]
@erikbern any plans to use interactive plotting like [Altair](https://altair-viz.github.io/) in python. Would help with more algorithms and still all pythonic without much overhead

**Comment by erikbern** [comment]
I'd be open to doing that. If you have time, feel free to give it a shot and submit a pull request!
